FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -713,11 +719,27 @@ extension ComposeViewController {
     }
 
     private func recipientsNode() -> RecipientEmailsCellNode {
-        RecipientEmailsCellNode(recipients: recipients.map(RecipientEmailsCellNode.Input.init))
+        return RecipientEmailsCellNode(
+            recipients: recipients.map(RecipientEmailsCellNode.Input.init),
+            height: calculatedRecipientsPartHeight ?? minRecipientsPartHeight
+        )
+            .onLayoutHeightChanged { [weak self] layoutHeight in
+                guard let self = self, self.calculatedRecipientsPartHeight != layoutHeight else {
+                    return
+                }
+
+                if layoutHeight > self.minRecipientsPartHeight {
+                    self.calculatedRecipientsPartHeight = layoutHeight
+                } else if layoutHeight > 0 {
+                    self.calculatedRecipientsPartHeight = layoutHeight
+                }
```

Comment:
Do we need both checks? We can leave only `if layoutHeight > 0` as `minRecipientsPartHeight` is bigger than 0 too.